src/Playwright/Program.cs

```diff
@@ -47,8 +47,7 @@ public int Run(string[] args)
             }
             catch (Exception)
             {
-                Console.WriteLine("Microsoft.Playwright is missing required assets. Please ensure to build your project before running.");
-                return 1;
+                return PrintError("Microsoft.Playwright build was found, but is missing required assets. Please ensure to build your project before running Playwright tool.");
```

Comment:
Not sure what `build was found` means in this context.  Maybe package?
```suggestion
                return PrintError("Microsoft.Playwright package was found, but is missing required assets. Ensure your project is built, before running Playwright tool.");
```

Reply:
assembly?